python-package/lightgbm/basic.py

```diff
@@ -487,7 +491,7 @@ def predict(self, data, num_iteration=-1,
             preds, nrow = self.__pred_for_np2d(data, num_iteration, predict_type)
         elif isinstance(data, list):
             try:
-                data = np.array(data)
+                data = np.array(data, copy=False)
```

Comment:
This is right after a check for `isinstance(data, list)`. The `copy=False` doesn't seem to make sense here.

Reply:
@dnouri Thanks! Fixed.